### ISSUE TITLE
How to save weights of part of a network in TF2.0?

### ISSUE THREAD

**Issue by yxchng** [tensorflow/tensorflow#36231]
TF provides `save_weights` API which saves the weights of a network. However, it does not allow one to save only part of the network. For instance, I have 

```
class X(Model):
    ...
    group1 = [layer_a, layer_b]
    group2 = [...]

model = X()
```

and I just want to save the weights of `group1`. PyTorch has the `ModuleList` API which is able to turn `group1` into a subnetwork and allows one to just call `X.group1.save_weights(...)`. What about TF? How to do so in TF2.0?

**Comment by ravikyram** [comment]
@yxchng 
Can you please go through the [link](https://www.tensorflow.org/tutorials/keras/save_and_load) ,[link2](https://www.tensorflow.org/guide/keras/save_and_serialize)and see if it helps you. Thanks!

**Comment by yxchng** [comment]
@ravikyram There is no mention of saving part of a model in these two links. Have you really read them?

**Comment by gowthamkpr** [comment]
Please go through this [issue](https://stackoverflow.com/questions/39450046/save-specific-weights-in-tensorflow) incombination with this [doc](https://www.tensorflow.org/api_docs/python/tf/compat/v1/train/Saver) and let me know if it helps. Thanks!

**Comment by tensorflowbutler** [comment]
It has been 15 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by gowthamkpr** [comment]
@yxchng Are you still facing the issue?

**Comment by gowthamkpr** [comment]
@yxchng You can actually try to save weights of few select layers as saving a part of the network is not possible.

**Comment by gowthamkpr** [comment]
Closing this issue as it has been inactive for more than 3 weeks. Please add additional comments and we can open the issue again. Thanks!
